src/mcts/search.cc

```diff
@@ -858,8 +858,7 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend(
     // Either terminal or unexamined leaf node -- the end of this playout.
     if (!node->HasChildren()) {
       if (node->IsTerminal()) {
-        IncrementNInFlight(node, search_->root_node_, collision_limit - 1);
-        return NodeToProcess::TerminalHit(node, depth, collision_limit);
+        return NodeToProcess::TerminalHit(node, depth, 1);
```

Comment:
Can you explain a bit? If I understand correctly this change affects the initialization of multivisit to 1, which in turn affects the calculation within FinalizeScoreUpdate to properly take Terminal evaluations to 1, correct? 
Would this change only have an affect when minibatchsize > 1 or is that not related at all?